core/controllers/reader.py

```diff
@@ -213,6 +213,25 @@ def get(self, exploration_id):
 
         exploration_rights = rights_manager.get_exploration_rights(
             exploration_id, strict=False)
+
+        # Retrieve all classifiers for the exploration.
+        state_classifier_mapping = {}
+        classifier_training_jobs = (
+            classifier_services.get_classifier_training_jobs(
+                exploration_id, exploration.version, exploration.states))
+        for index, state_name in enumerate(exploration.states):
+            if classifier_training_jobs[index] == None:
+                state_classifier_mapping[state_name] = None
+            else:
+                # Not making this a multi-get since we are deprecating
```

Comment:
@seanlip We thought we'll do this first since it blocks Prasanna's PR on frontend Code classifier service and his GSOC project in general. ClassifierDataModel deprecation wasn't technically a blocker and more of an optimisation.  But, if you're certain that we can't do GETs in a for loop, I think it might be better to get the deprecation done with.